src/node/rpc/member_frontend.h

```diff
@@ -831,10 +831,14 @@ namespace ccf
            {in.quote_info},
            in.public_encryption_key,
            NodeStatus::TRUSTED,
-           get_next_reconfiguration_id(network, ctx.tx),
            std::nullopt,
            ds::to_hex(in.code_digest.data)});
 
+        kv::NetworkConfiguration nc =
```

Comment:
We should move this to `add_node()`, only if the status is `NodeStatus::TRUSTED` so that we don't forget some cases where nodes are added.